src/sif/recost.cc

```diff
@@ -26,7 +25,6 @@ void recost_forward(baldr::GraphReader& reader,
                     float source_pct,
                     float target_pct,
                     const baldr::TimeInfo& time_info,
-                    const bool invariant,
```

Comment:
removing this is a breaking API change. but we can su pport what you want to do in another way. see my larger pr comment for details or let me know if you want to meet to talk about it in more detail.

Reply:
fixed it, added an option for using invariant parameter on path postprocessing.